### ISSUE TITLE
Create subdirectories per version

### ISSUE THREAD

**Issue by iemejia** [patricklucas/docker-flink#1]
We should maybe create a subdirectory for each flink-version, so instead of current
```
1.1/hadoop24-scala_2.10-alpine
...
1.2/hadoop24-scala_2.10-alpine
...
```
we may better have:
```
1.1.4/hadoop24-scala_2.10-alpine
...
1.2.0/hadoop24-scala_2.10-alpine
...
```

**Comment by patricklucas** [comment]
Judging from all the other official repos I looked at, top level directories are for maintained releases, and the Dockerfiles within them are kept up to date with the latest patch version within the releases.

For example, [httpd|https://github.com/docker-library/httpd] has top level directories for 2.2 and 2.4, and the Dockerfile within the 2.4 directory references version 2.4.25, the latest 2.4 version.

Admittedly, I'm not totally up to date on which versions of Flink are considered "supported"—it may actually just be 1.2, but I included 1.1 to ensure the scripts here supported multiple active releases.

**Comment by iemejia** [comment]
Ok your arguments convinced me, so let's keep it like it is. I think both 1.1 and 1.2 are officially supported for the moment and yes I agree that we should maintain only the currently maintained versions.
